File: tests/benchmark_loop.dart
main() {
	 var sw = new Stopwatch();
	 var iter = 0;
	 sw.start();
	 while (iter%128 != 0 || sw.elapsed.inSeconds < 1) {
		iter++;
	 }
	 sw.stop();
	 print("$iter iterations");
}

File: tests/benchmark_loop.java
class BenchmarkLoop {
	public static void main(String[] args) {
		long iter = 0;
		start = System.currentTimeMillis();
		while (iter % 128 != 0 || (System.currentTimeMillis() - start) < 1000) {
			iter++;
		}
		System.out.println(iter + " interations");
	}
}
